src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/config/KafkaTopicsGrpcConfig.java

```diff
@@ -0,0 +1,416 @@
+/* Copyright 2019 Telstra Open Source
```

Comment:
Not requesting a change, but just some thoughts: copyright since 2019 includes copyright since 2024 😄, but for the new files it's better to use the current year? The caveat is that for old files this new claim could be understood as it removes the claim from the previous version or the opposite, it's been claimed in 19, but the edit in 24 goes without the copyright statement. To resolve ambiguity, some companies put two years: Copyright 2019,2024, the year of the first publication and when the last edit has been made.